jetstream/outcomes/firefox_desktop/crlite.toml

```diff
@@ -48,11 +53,18 @@ select_expression = """
 """
 friendly_name = "OCSP usage"
 description = "Number of revocation checks performed using OCSP."
+depends_on = ["ocsp_use_counter", "revocation_check_counter"]
```

Comment:
```suggestion
```

this can go I guess with the `_ratio` metric

Reply:
Ah whoops, thanks!